src/charm.py

```diff
@@ -146,6 +146,10 @@ def parse_client_config_args(self):
             if key == "ssl-public-key":
                 value = parse_ssl_arg(value)
 
+            if isinstance(value, (int, float)):
```

Comment:
Maybe just do `value = str(value)` since it can't be anything else right

Reply:
In our case, I guess so. We don't have any `boolean` types, which are the only ones that I thing could have some pitfalls.
I'll change it.

Reply:
Yeah I mean boolean types should still not be passed into Popen